test/integration/serverless/snapshots/error-node

```diff
@@ -1,631 +1,3 @@
-      "functionname:integration-tests-extension-XXXXXX-error-node",
-      "memorysize:1024",
-      "region:eu-west-1",
-      "resource:integration-tests-extension-XXXXXX-error-node",
-      "runtime:nodejs14.x",
-      "service:integration-tests-service",
-      "taga:valuea",
-      "tagb:valueb",
-      "tagc:valuec",
-      "tagd:valued",
-      "version:integration-tests-version"
-    ]
-  },
-  {
-    "distributions": null,
-    "dogsketches": [],
-    "metric": "aws.lambda.enhanced.response_latency",
-    "tags": [
-      "account_id:425362996713",
-      "architecture:XXX",
-      "aws_account:425362996713",
-      "cold_start:true",
-      "dd_extension_version:123",
-      "env:integration-tests-env",
-      "function_arn:arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-error-node",
-      "functionname:integration-tests-extension-XXXXXX-error-node",
-      "memorysize:1024",
-      "region:eu-west-1",
-      "resource:integration-tests-extension-XXXXXX-error-node",
-      "runtime:nodejs14.x",
-      "service:integration-tests-service",
-      "taga:valuea",
-      "tagb:valueb",
-      "tagc:valuec",
-      "tagd:valued",
-      "version:integration-tests-version"
-    ]
-  },
-  {
-    "distributions": null,
-    "dogsketches": [],
-    "metric": "aws.lambda.enhanced.runtime_duration",
-    "tags": [
-      "account_id:425362996713",
-      "architecture:XXX",
-      "aws_account:425362996713",
-      "cold_start:false",
-      "dd_extension_version:123",
-      "env:integration-tests-env",
-      "function_arn:arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-error-node",
-      "functionname:integration-tests-extension-XXXXXX-error-node",
-      "memorysize:1024",
-      "region:eu-west-1",
-      "resource:integration-tests-extension-XXXXXX-error-node",
-      "runtime:nodejs14.x",
-      "service:integration-tests-service",
-      "taga:valuea",
-      "tagb:valueb",
-      "tagc:valuec",
-      "tagd:valued",
-      "version:integration-tests-version"
-    ]
-  },
-  {
-    "distributions": null,
-    "dogsketches": [],
-    "metric": "aws.lambda.enhanced.runtime_duration",
-    "tags": [
-      "account_id:425362996713",
-      "architecture:XXX",
-      "aws_account:425362996713",
-      "cold_start:true",
-      "dd_extension_version:123",
-      "env:integration-tests-env",
-      "function_arn:arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-error-node",
-      "functionname:integration-tests-extension-XXXXXX-error-node",
-      "memorysize:1024",
-      "region:eu-west-1",
-      "resource:integration-tests-extension-XXXXXX-error-node",
-      "runtime:nodejs14.x",
-      "service:integration-tests-service",
-      "taga:valuea",
-      "tagb:valueb",
-      "tagc:valuec",
-      "tagd:valued",
-      "version:integration-tests-version"
-    ]
-  }
-]
+{
+  "error": "normalization raised exception"
+}
```

Comment:
This change seems wrong to me.

Reply:
You're right! This was PEBCAK I had already fixed, but then... this file was missing from my changeset when I committed... I've fixed it now...!